v6.2 Important upgrade notes (#5282)
mattermost/docs: source/upgrade/important-upgrade-notes.rst

+----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
+====================================================+==================================================================================================================================================================+

> | v6.2                                               | Channel results in the channel switcher will include channels from other teams as well as private channels. For users using Bleve or Elasticsearch for           |

Reviewer: ```suggestion
| v6.2                                               | Channel results in the channel menu will include channels from other teams as well as private channels. Customers using Bleve or Elasticsearch for           |
```

Reviewer: Recommend moving away from "switch" and "switcher" terminology, as per Marketing.

> | v6.2                                               | Channel results in the channel switcher will include channels from other teams as well as private channels. For users using Bleve or Elasticsearch for           |

Reviewer 2: I should clarify that this is only for autocomplete, and not search. Since search is not affected by channels.

> | v6.2                                               | Channel results in the channel menu will include channels from other teams as well as private channels. Customers using Bleve or Elasticsearch for               |

Reviewer 3: I don't think the cross team autocomplete was done yet? My understanding is that the front end is still outstanding? cc// @agnivade

Reviewer 3: Is a reindex still required even for just private channel autocomplete?

Reviewer 2: My apologies, I misread it quickly, yes cross team autocomplete isn't done. Only private channel autocomplete is.

Yes, reindex is still required even for private channel autocomplete.

Author: I edited the text.

Reviewer 2: Hey @amyblais - This is specifically for the channel autocomplete, not the channel switcher. Channel switcher always showed private channels.

> |                                                    | autocomplete/search will have to reindex their data to get the new results. Since this can take a long time, it is suggested to disable autocomplete and run     |

Reviewer: ```suggestion
|                                                    | autocomplete/search will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running     |
```

Reviewer: My proposed updates may break the table from displaying properly -- it was challenging to manage the table syntax with these edits.
